plugins/unifying/fu-plugin-unifying.c

```diff
@@ -278,6 +278,17 @@ gboolean
 fu_plugin_startup (FuPlugin *plugin, GError **error)
 {
 	FuPluginData *data = fu_plugin_get_data (plugin);
+
+	/* check the kernel has CONFIG_HIDRAW */
+	if (!g_file_test ("/sys/class/hidraw", G_FILE_TEST_IS_DIR)) {
+		g_set_error_literal (error,
+				     FWUPD_ERROR,
+				     FWUPD_ERROR_NOT_SUPPORTED,
+				     "no CONFIG_HIDRAW support");
```

Comment:
I think it's clearer to indicate the problem lies in the kernel (An average passer by won't know what CONFIG_HIDRAW means)

Reply:
"no kernel support for CONFIG_HIDRAW" ? I really wanted to have something googleable, and in practice the only people hitting this will have built their own kernel themselves.

Reply:
Yeah that's better.